src/components/templates/Preplaya/VenuePreview.scss

```diff
@@ -0,0 +1,55 @@
+.playa-venue-preview-container {
+  padding-bottom: 30px;
+
+  & > .header {
+    height: 400px;
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    object-fit: cover;
+    margin: 0 -15px;
+    border-radius: 22px;
+
+    .title-container {
+      display: flex;
+      width: 100%;
+      padding: 0 22px;
+      align-items: flex-end;
+
+      .host-icon {
+        width: 25%;
+        border-radius: 22px;
```

Comment:
will this always make it a circle? what if 25% > 44px?

Reply:
It's not a circle on the rough design: https://trello.com/c/YGaHIjm2/474-aau-when-i-click-on-a-venue-on-the-playa-i-see-the-party-modal

We can revisit when we get the final designs.